server/routes/induction/create/index.ts

```diff
@@ -89,5 +92,12 @@ export default (router: Router, services: Services) => {
     router.get('/prisoners/:prisonNumber/create-induction/has-worked-before', [
       workedBeforeCreateController.getWorkedBeforeView,
     ])
+
+    router.get('/prisoners/:prisonNumber/create-induction/reasons-not-to-get-work', [
+      asyncMiddleware(reasonsNotToGetWorkCreateController.getReasonsNotToGetWorkView),
```

Comment:
There is a unlikely, but possible bug where if a user session expires while creating an induction and tries to log back in the DTO held in the session object is now undefined. This then causes the application to crash when trying to access properties from the now undefined DTO.

I considered adding error handling for this in the controller itself, but rare edge case exceptions like this should probably just be caught. Currently we don't catch errors that are thrown in the controller which means any unexpected error thrown will cause the application to crash. In live this would cause pods to crash and restart when unexpected errors occur.

`asyncMiddleware` catches any controller errors and allows them to be passed on to the error handler middleware which will show an error page to the user. I think we should probably use this everywhere, but just added here for now as a suggestion.